packages/terra-clinical-data-grid/package.json

```diff
@@ -37,13 +37,13 @@
     "prop-types": "^15.5.8",
     "react-draggable": "^3.0.5",
     "resize-observer-polyfill": "^1.4.1",
-    "terra-button": "^2.0.1",
```

Comment:
This leaves react-intl as the only lerna warning (2.7.0 vs 2.4.0).

Reply:
Lets go ahead and match those as well. It looks like onset-picker is using 2.4.0. We can bump it to 2.7.0 to match data-grid or fall back to 2.0.0

Reply:
It was set to 2.7.0, and falling back to 2.0.0 also gives a hoist warning. Setting it to 2.4.0 seems to work.

Reply:
We would need to bump onset picker to 2.7.0 to match the data-grid's specification. I"m good with either way; the latest will be installed regardless.

Reply:
I went ahead and set both onset picker and data grid to 2.7.0: [b34fb92](https://github.com/cerner/terra-clinical/pull/448/commits/b34fb9201be7da2bdc94c4c4733f804eb77a545d)